### PR TITLE
Add support for msys2 platform (#291)

### PR DESCRIPTION
**Pull request by vermilion-bird**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Plug 'iamcco/markdown-preview.nvim', { 'do': { -> mkdp#util#install() }, 'for':
 
 
 " If you have nodejs and yarn
-Plug 'iamcco/markdown-preview.nvim', { 'do': 'cd app && yarn install'  }
+Plug 'iamcco/markdown-preview.nvim', { 'do': 'cd app && yarn install && npm install'  }
```
Comment (iamcco) [inline README.md:45]: `&& npm install` is no need.

Comment (iamcco) [reply README.md:45]: It should be `yarn install` or `npm install`

**Comment by iamcco** [comment]
Thanks

File: README.md (hunk 1 above)
Comment (vermilion-bird) [reply README.md]: Thanks. I tested it again and found that only ```npm install``` is need for msys2 platform. Now I have modified this pull request.

**Pull request merged**